### PR TITLE
new recipe: indextools

### PR DESCRIPTION
**Pull request by matthdsm**
----

Please read the [guidelines for Bioconda recipes](https://bioconda.github.io/contributor/guidelines.html) before opening a pull request (PR).

* If this PR adds or updates a recipe, use "Add" or "Update" appropriately as the first word in its title.
* New recipes not directly relevant to the biological sciences need to be submitted to the [conda-forge channel](https://conda-forge.org/docs/) instead of Bioconda.
* PRs require reviews prior to being merged. Once your PR is passing tests and ready to be merged, please issue the `@BiocondaBot please add label` command.
* Please post questions [on Gitter](https://gitter.im/bioconda/Lobby) or ping `@bioconda/core` in a comment.

<details>
  <summary>Please use the following BiocondaBot commands:</summary>

Everyone has access to the following BiocondaBot commands, which can be given in a comment:

<table>
  <tr>
    <td><code>@BiocondaBot please update</code></td>
    <td>Merge the master branch into a PR.</td>
  </tr>
  <tr>
    <td><code>@BiocondaBot please add label</code></td>
    <td>Add the <code>please review & merge</code> label.</td>
  </tr>
  <tr>
    <td><code>@BiocondaBot please fetch artifacts</code></td>
    <td>Post links to CI-built packages/containers. <br />You can use this to test packages locally.</td>
  </tr>
</table>

For members of the Bioconda project, the following command is also available:

<table>
  <tr>
    <td><code>@BiocondaBot please merge</code></td>
    <td>Upload built packages/containers and merge a PR. <br />Someone must approve a PR first! <br />This reduces CI build time by reusing built artifacts.</td>
  </tr>
</table>

Also, the bot watches for comments from non-members that include `@bioconda/<team>` and will automatically re-post them to notify the addressed `<team>`.

</details>

### CODE REVIEW

File: recipes/indextools/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: bfba993dfb0ea2cf00b6f8abdcc5972e517ff9ae727e3da984db8e880a7240b3
+
+build:
+  number: 0
+  entry_points:
```
Comment (matthdsm) [inline recipes/indextools/meta.yaml:14]:
```suggestion
  noarch: python
  entry_points:
```

**Comment by matthdsm** [comment]
@BiocondaBot please add label

**Comment by matthdsm** [comment]
pending https://github.com/conda-forge/staged-recipes/pull/22271
and https://github.com/conda-forge/staged-recipes/pull/22270

**Review by apeltzer — APPROVED** [review]

**Review by maxulysse — APPROVED** [review]

File: recipes/indextools/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+    - indextools = indextools.console:indextools
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - autoclick >=0.6.1
+    - loguru >=0.3.2
+    - ngsindex >=0.1.7
+    - pip
```
Comment (matthdsm) [inline recipes/indextools/meta.yaml:24]:
```suggestion
    - pip
    - poetry
```

File: recipes/indextools/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+    - ngsindex >=0.1.7
+    - pysam >=0.15
+    - python
+    - xphyle >=4.1.3
+
+test:
+  imports:
+    - cgranges
+    - indextools
+    - indextools.console
```
Comment (matthdsm) [inline recipes/indextools/meta.yaml:41]:
```suggestion
```

File: recipes/indextools/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+    - loguru >=0.3.2
+    - ngsindex >=0.1.7
+    - pysam >=0.15
+    - python
+    - xphyle >=4.1.3
+
+test:
+  imports:
+    - cgranges
+    - indextools
```
Comment (matthdsm) [inline recipes/indextools/meta.yaml:40]:
```suggestion
```

**Pull request closed without merging**